### PR TITLE
Pants 2.23.0 release blog post.

### PR DESCRIPTION
**Pull request by benjyw**
Plus some formatting nits in the previous release post.

### CODE REVIEW

File: blog/2024-09-10-pants-2-22/index.mdx [hunk 1]
```diff
@@ -8,14 +8,10 @@
 # Pants 2.22.0 is released!
 
 <CaptionedImg src={require("./splash.jpeg").default}>
-  Photo by{" "}
```
Comment (benjyw) [inline blog/2024-09-10-pants-2-22/index.mdx:11]: This was the copy-pasted HTML that unsplash recommends, which our formatter reformatted in a way that broke the rendering (spread it over multiple lines for no reason). Switching to markdown renders correctly after formatting the source.

**Comment by benjyw** [comment]
Rendered: 

localhost_3000_blog_2024_11_20_pants2-23

File: blog/2024-11-20-pants2-23/index.mdx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+The previous release, 2.22.0, suffered from delays in its release cycle. So this 2.23.0 release is a relatively fast follow, and does not introduce major upheavals.
+
+A couple of administrative reminders:
+
+- The annual [Pants user survey](https://www.pantsbuild.org/blog/2024/10/08/user-survey-2024) is open through the end of 2024. The results will be used to assist the Pants maintainers and contributors to better understand areas where Pants may need work, so your feedback is vital!
+- We're very proud that Pants now has its first [sponsors](https://www.pantsbuild.org/sponsors)! We are very grateful to [Klayvio](https://www.klaviyo.com/) and [Normal Computing](https://normalcomputing.ai/) for their Platinum tier support, and to [Continua](https://www.continua.ai/) and Stormfish for their Silver tier support.
```
Comment (tdyas) [inline blog/2024-11-20-pants2-23/index.mdx:26]: Should Stormfish get a link?

File: blog/2024-11-20-pants2-23/index.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  Sponsoring is a way for companies that use Pants to support the project while engaging more closely with the community. Want to support Pants? Find out more about our [sponsorship opportunities](https://www.pantsbuild.org/sponsorship) and the associated benefits at each tier.
+
+_Highlights in [2.23](https://github.com/pantsbuild/pants/tree/2.23.x/src/python/pants/notes/2.23.x.md) include_:
```
Comment (tdyas) [inline blog/2024-11-20-pants2-23/index.mdx:29]: What about the fact that 2.23 "completed" the in-workspace execution support? Or should we leave that to the forthcoming blog post on in-workspace execution?

**Review by tdyas — APPROVED** [review]

File: blog/2024-11-20-pants2-23/index.mdx (hunk 2 above)
Comment (benjyw) [reply blog/2024-11-20-pants2-23/index.mdx:26]: They don't have a website yet. Am waiting on it.

File: blog/2024-11-20-pants2-23/index.mdx (hunk 3 above)
Comment (benjyw) [reply blog/2024-11-20-pants2-23/index.mdx:29]: I took these from the highlights in the release notes. Happy to add a sentence if you have one.

Comment (tdyas) [reply blog/2024-11-20-pants2-23/index.mdx:29]: It is not just plugins. `experimental_workspace_environment` exposes the functionality in `shell_command` and `adhoc_tool`.  How about this:

> Experimental support for in-workspace execution via a new `experimental_workspace_environment` which allows running processes in your repository without a sandbox via `shell_command` and `adhoc_tool`. This is useful for interfacing with other tools (such as another build system) which expect to run directly in the repository.

Comment (tdyas) [reply blog/2024-11-20-pants2-23/index.mdx:29]: Hmm 2.22 already announced it.

Comment (tdyas) [reply blog/2024-11-20-pants2-23/index.mdx:29]: 2.23 just "completes" the in-workspace execution feature in a way, it was introduced in 2.22.

Disregard my comment then.

File: blog/2024-11-20-pants2-23/index.mdx
Comment (cognifloyd) [inline blog/2024-11-20-pants2-23/index.mdx:1]: The other file is in the `<date>-pants-2-22`, so should this be in `<date>-pants-2-23` instead? (add a dash between pants and 2)

Comment (benjyw) [reply blog/2024-11-20-pants2-23/index.mdx:1]: Ah yes, I think that was a typo

**Review by cognifloyd — APPROVED** [review]

**Pull request merged**